x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -408,7 +401,7 @@ export const SEARCH_RULES = i18n.translate(
 export const SEARCH_PLACEHOLDER = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.allRules.searchPlaceholder',
   {
-    defaultMessage: 'e.g. rule name',
+    defaultMessage: 'Search by rule name, index pattern, or MITRE ATT&CK tactic or technique',
```

Comment:
Also noticed a small UX item here: if a user copies the whole tactic/technique from Rule Details, ie. `Defense Evasion (TA0005)` and pastes that in they won't get any results since `id`/`name` are independent fields -- they would need to search ` Defense Evasion` or `TA0005`. Not too much we could do here outside updating the placeholder text on the search bar to include an example maybe?

Reply:
Yeah, I agree 👍  Providing an example would make this UI a bit friendlier to our users.